api/src/main/java/io/strimzi/api/kafka/model/KafkaUserSaslPlainClientAuthentication.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
See the comment above - I think you should not do this change in this PR (and it should not be related).

Reply:
Same as above, thanks for the advise.

Reply:
code updated, please review ..